src/parquet/util/test-common.h

```diff
@@ -72,6 +75,49 @@ static inline vector<bool> flip_coins(size_t n, double p) {
   return draws;
 }
 
+const char* data_dir = std::getenv("PARQUET_TEST_DATA");
+
+class TestFileInfo {
```

Comment:
maybe struct here (for POD structs you can omit the `_` on members variables https://google.github.io/styleguide/cppguide.html#Variable_Names)?